src/chainstate/coordinator/mod.rs

```diff
@@ -401,7 +403,11 @@ pub fn get_reward_cycle_info<U: RewardSetProvider>(
 
         let reward_cycle_info = {
             let ic = sort_db.index_handle(sortition_tip);
-            ic.get_chosen_pox_anchor(&parent_bhh, &burnchain.pox_constants)
+            ic.get_chosen_pox_anchor(
+                &parent_bhh,
+                &burnchain.pox_constants,
+                burnchain.first_block_height,
```

Comment:
This should be equal to `sort_db.first_block_height`, and moreover, `sort_db.first_block_height` should be available in `ic.context`.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2233/commits/fd507c59944e9f7d373b364b2beceea6cf00315a